Makefile

```diff
@@ -1,35 +1,29 @@
-.PHONY: all-checks
+.PHONY: all-checks check coverage docs format package requirements test
+
 all-checks: requirements coverage check docs package
 
-.PHONY: requirements
 requirements:
 	python3 -m pip install -r requirements/development.txt
 	python3 -m pip install --editable .
 
-.PHONY: check
 check:
 	black --check .
-	flake8 --exclude .venv,venv,docs/conf.py .
+	flake8 more_itertools tests
```

Comment:
Let's be more specific in targeting code for checks and assuming nothing out is of any interest.